app/cdap/components/ParsingDrawer/Components/ParsingHeaderActionTemplate/index.tsx

```diff
@@ -38,15 +35,16 @@ export default function(props) {
       } catch (e) {
         props.setErrorOnTransformation({
           open: true,
-          message: T.translate('features.WranglerNewParsingDrawer.importSchemaErrorMessage'),
+          message: 'Imported schema is not a valid Avro schema',
```

Comment:
message should come from translation

Reply:
done

Reply:
change the hierachy from features.WranglerNewParsingDrawer to 
features.WranglerNewUI.WranglerNewParsingDrawer

Reply:
done